distribution/lib/Standard/Base/0.0.0-dev/src/Error/Common.enso

```diff
@@ -761,6 +761,11 @@ type Time_Error
        - error_message: The message for the error.
     Time_Error_Data error_message
 
+    ## PRIVATE
+    epoch_start : Time_Error
+    epoch_start = Time_Error_Data "Epoch start underflow"
```

Comment:
I'd expect a more detailed message, so that the user knows what has happened. Can we have a short version of the explanation from the `Date_Time.epoch_start` docstring?